phpunit.xml.dist

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- PHPUnit configuration file with new format for PHPUnit 9.5+ -->
+<!-- PHPUnit configuration file with new format for current PHPUnit -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd"
          bootstrap="vendor/autoload.php"
          cacheResult="false"
-         colors="true"
-         convertDeprecationsToExceptions="true">
+         colors="true">
```

Comment:
I can approve this pull request, but I think we should mention that we might loose future visibility of deprecation notices. I looked a bit further into this.

In the [changelog for PHPUnit 10](https://github.com/sebastianbergmann/phpunit/blob/10.0.0/ChangeLog-10.0.md) it says the following:
> Removed the convertDeprecationsToExceptions attribute on the <phpunit> XML configuration element as well as the feature it used to control

So it seems we can't use `convertDeprecationsToExceptions` anymore, but there is a new option called `displayDetailsOnTestsThatTriggerDeprecations`. Not sure if this will have the same effect tho.

One solution to this could be to add another `phpunit.xml` specifically for PHPUnit 9 (PHP >= 7.3), but this feels kind of dirty, what do you think?

Reply:
Agree that we'd want to keep this functionality, fortunately looks like this might be re-introduced in PHPUnit 10.1 again via https://github.com/sebastianbergmann/phpunit/issues/5196

We can keep this open until the upstream ticket is resolved, looks like we'll need to use separate XML configurations for PHPUnit 10, PHPUnit 9 and older in either case to support the current range of PHP versions. Not a fan of this, but fine, if that's what it takes.